content/projects/understanding-loops/_index.md

```diff
@@ -114,50 +145,33 @@ Then it means that the person is buying 2 hamsters at a price of R20 each. So th
 
 Please note that one person can have multiple baskets. If you look at the data, you'll see that tshepo@umuzi.org has 4 baskets. Two have been delivered, one has been paid for (so he's awaiting delivery) and one is open (so he's setting up his next order).
 
-## Write some functionality
-
-### Important notes for Javascript solutions
-
-#### 1. Destructuring
-
-In Javascript it is best practice to use destructuring when passing arguments to a function. You can learn about destructuring here: [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
-
-Please make sure you use destructuring for all your functions.
-
-```
-getCustomerBaskets(shoppingBaskets, "sine@umuzi.org"); // DON'T DO THIS
-getCustomerBaskets("sine@umuzi.org", customerBaskets); // DON'T DO THIS EITHER
+## Working with json files 
 
-// The problem with the above code is that if you get the order of the parameters wrong, then things will break. Your functions should work like this instead:
+JSON is a lovely data format because it is easy for humans to read, and machines like it too. 
 
-function getCustomerBaskets({ shoppingBaskets, email }) {...}
+If you are working in **Python** then you can turn the json file into a data structure in your code by making use of the [json package](https://docs.python.org/3/library/json.html).
 
-getCustomerBaskets({shoppingBaskets, email}) //DO THIS
-```
+In JavaScript you can simply `require` a JSON file and then it'll get converted into the appropriate data-structure.
 
```

Comment:
Java is way more complicated.  Looking at learner projects that were recently marked as competent, everyone did different things. I'm not sure what best practice is. Can you please make a recommendation here?

Reply:
I added the java section for this, please check it out and let me know what you think